### DISCUSSION TITLE
Change repository url

### DISCUSSION THREAD

**Discussion by KeiShadow** [codemagic-ci-cd/codemagic-docs#924] [Q&A] [answered]
Hello, im woner of loono app and i cant change repository url.
I cant write to text input field.

Could you help me please<

**Comment by K-Codemagic** [comment, marked as answer]
Hi, is your migrated to a different repository? Can you please share the new and old URL changes?

If the repository is added via GitHub, Bitbucket or GitLab integration, click the Update repository URL button. Codemagic will attempt to automatically update the URL based on the repository ID.

If the repository is added from a URL, click Change settings under Repository access settings. This will allow you to modify the repository URL, change username or password or update the SSH key. You can also generate a new private key in the settings section. Once you have changed the relevant settings, click Update access settings. Codemagic will verify that it has access to the repository using the updated details before saving the settings.

**Reply by KeiShadow** [reply]
Yeah we just migrate from this repo: https://github.com/cesko-digital/loono to this repository: https://github.com/Loono-cz/loono

I cant find any Repository access settings section.
And if i click on Update repository URL button, it showing me this message:

Cannot access repositories
Not Found

**Reply by K-Codemagic** [reply]
Hi, it didn't work as it has been moved to a different organization. Can you please add the new url app in Codemagic, and share the old and new app ids, so I can fix it on the backend to keep your old configuration and workflow settings intact?

**Reply by KeiShadow** [reply]
Nice i create new application.
Now this is old app: https://codemagic.io/app/60c62c436d84ba547266cc1d
And this is new one: https://codemagic.io/app/630f67192923d843b370bf5c

Thank you very much.

**Reply by K-Codemagic** [reply]
Hi, I have made the changes in the back end. It has been updated now from https://github.com/cesko-digital/loono -> https://github.com/Loono-cz/loono. You can now check your old app which would have the updated URL and confirm. Also delete the new app, if not needed.

Thanks, and have a nice day.

**Reply by KeiShadow** [reply]
Thank you very much ! Nice support ✌️
